stories/features/sync/enabledContent.tsx

```diff
@@ -0,0 +1,155 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Components
+import Card from '../../../src/components/layout/card'
+import Toggle from '../../../src/components/formControls/toggle'
+import Button from '../../../src/components/buttonsIndicators/button'
+import Table, { Cell, Row } from '../../../src/components/dataTables/table'
+
+// Feature-specific components
+import {
+  Grid,
+  FlexColumn,
+  SwitchLabel,
+  Label,
+  Paragraph,
+  SectionBlock,
+  SubTitle
+} from '../../../src/features/sync'
+
+// Modals
+import SyncANewDeviceModal from './modals/syncNewDevice'
+import ResetSyncModal from './modals/resetSync'
+
+// Utils
+import locale from './page/fakeLocale'
+
+interface SyncEnabledContentState {
+  syncANewDeviceModal: boolean
```

Comment:
Same nit re use of the word 'modal' here - stateful props passed from caller could be a little more semantic